### ISSUE TITLE
Running cert errors

### ISSUE THREAD

**Issue by jdnielss** [sherifabdlnaby/elastdocker#76]
hi, i wan't to running ELK using this repository.

I follow the tutorial from this readme, but I got an error when "make setup" in pulling cert.

configuration is default from this repo

![image](https://user-images.githubusercontent.com/28420302/174856962-80dd38df-5f84-46f4-a332-b0f0de4909bd.png)

```
[2022-06-21T17:10:02.175+00:00][INFO ][plugins-service] Plugin "cloudSecurityPosture" is disabled.
[2022-06-21T17:10:02.242+00:00][INFO ][plugins-service] Plugin "metricsEntities" is disabled.
[2022-06-21T17:10:02.360+00:00][INFO ][http.server.Preboot] http server running at https://0.0.0.0:5601
[2022-06-21T17:10:02.399+00:00][INFO ][plugins-system.preboot] Setting up [1] plugins: [interactiveSetup]
[2022-06-21T17:10:02.434+00:00][WARN ][config.deprecation] The default mechanism for Reporting privileges will work differently in future versions, which will affect the behavior of this cluster. Set "xpack.reporting.roles.enabled" to "false" to adopt the future behavior before upgrading.
[2022-06-21T17:10:02.752+00:00][INFO ][plugins-system.standard] Setting up [117] plugins: [translations,monitoringCollection,licensing,globalSearch,globalSearchProviders,features,mapsEms,licenseApiGuard,usageCollection,taskManager,telemetryCollectionManager,telemetryCollectionXpack,kibanaUsageCollection,share,embeddable,uiActionsEnhanced,screenshotMode,screenshotting,banners,newsfeed,fieldFormats,expressions,eventAnnotation,dataViews,charts,esUiShared,bfetch,data,savedObjects,presentationUtil,expressionShape,expressionRevealImage,expressionRepeatImage,expressionMetric,expressionImage,customIntegrations,home,searchprofiler,painlessLab,grokdebugger,management,watcher,advancedSettings,spaces,security,savedObjectsTagging,reporting,lists,fileUpload,ingestPipelines,encryptedSavedObjects,dataEnhanced,cloud,snapshotRestore,telemetry,licenseManagement,eventLog,actions,alerting,triggersActionsUi,transform,stackAlerts,ruleRegistry,savedObjectsManagement,console,controls,graph,fleet,indexManagement,remoteClusters,crossClusterReplication,indexLifecycleManagement,visualizations,canvas,visTypeXy,visTypeVislib,visTypeVega,visTypeTimeseries,rollup,visTypeTimelion,visTypeTagcloud,visTypeTable,visTypeMetric,visTypeHeatmap,visTypeMarkdown,dashboard,dashboardEnhanced,expressionXY,expressionTagcloud,expressionPartitionVis,visTypePie,expressionMetricVis,expressionHeatmap,expressionGauge,visTypeGauge,dataViewFieldEditor,sharedUX,discover,lens,osquery,maps,dataVisualizer,ml,cases,timelines,sessionView,securitySolution,observability,uptime,infra,upgradeAssistant,monitoring,logstash,enterpriseSearch,apm,discoverEnhanced,dataViewManagement]
[2022-06-21T17:10:02.783+00:00][INFO ][plugins.taskManager] TaskManager is identified by the Kibana UUID: 11854da9-9ba1-4a72-a255-0df2d387cf40
[2022-06-21T17:10:03.222+00:00][INFO ][plugins.ruleRegistry] Installing common resources shared between all indices
[2022-06-21T17:10:04.428+00:00][INFO ][plugins.screenshotting.config] Chromium sandbox provides an additional layer of protection, and is supported for Linux Ubuntu 20.04 OS. Automatically enabling Chromium sandbox.
[2022-06-21T17:10:04.711+00:00][ERROR][elasticsearch-service] Unable to retrieve version information from Elasticsearch nodes. connect ECONNREFUSED 172.21.0.3:9200
[2022-06-21T17:10:05.805+00:00][INFO ][plugins.screenshotting.chromium] Browser executable: /usr/share/kibana/x-pack/plugins/screenshotting/chromium/headless_shell-linux_x64/headless_shell
```

**Comment by jdniels** [comment]
![image](https://user-images.githubusercontent.com/28420302/174857799-6d39137a-39d9-4353-a8fb-a0216b60253b.png)
and i run command docker-compose, i got error kibana error

**Comment by jdniels** [comment]
Solved, but i got more error 


```
{"type": "server", "timestamp": "2022-06-21T17:18:24,324Z", "level": "WARN", "component": "o.e.x.s.t.n.SecurityNetty4HttpServerTransport", "cluster.name": "elastdocker-cluster", "node.name": "elastdocker-node-0", "message": "received plaintext http traffic on an https channel, closing connection Netty4HttpChannel{localAddress=/172.21.0.2:9200, remoteAddress=/108.137.5.45:59596}", "cluster.uuid": "RKA4SIDNTyyN8nJIfj1ywA", "node.id": "AeumPgfnTXiqaN9ov8Srbw"  }
```


```
{"type": "server", "timestamp": "2022-06-21T17:17:19,258Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "elastdocker-cluster", "node.name": "elastdocker-node-0", "message": "version[8.2.0], pid[7], build[default/docker/b174af62e8dd9f4ac4d25875e9381ffe2b9282c5/2022-04-20T10:35:10.180408517Z], OS[Linux/4.14.281-212.502.amzn2.x86_64/amd64], JVM[Eclipse Adoptium/OpenJDK 64-Bit Server VM/18/18+36]" }
{"type": "server", "timestamp": "2022-06-21T17:17:19,276Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "elastdocker-cluster", "node.name": "elastdocker-node-0", "message": "JVM home [/usr/share/elasticsearch/jdk], using bundled JDK [true]" }
{"type": "server", "timestamp": "2022-06-21T17:17:19,277Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "elastdocker-cluster", "node.name": "elastdocker-node-0", "message": "JVM arguments [-Xshare:auto, -Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -Djava.security.manager=allow, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -XX:+ShowCodeDetailsInExceptionMessages, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Dlog4j2.formatMsgNoLookups=true, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.io=ALL-UNNAMED, -XX:+UseG1GC, -Djava.io.tmpdir=/tmp/elasticsearch-14268149124759273297, -XX:+HeapDumpOnOutOfMemoryError, -XX:+ExitOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Des.cgroups.hierarchy.override=/, -Xmx1024m, -Xms1024m, -Des.enforce.bootstrap.checks=true, -Dlog4j2.formatMsgNoLookups=true, -XX:MaxDirectMemorySize=536870912, -XX:G1HeapRegionSize=4m, -XX:InitiatingHeapOccupancyPercent=30, -XX:G1ReservePercent=15, -Des.path.home=/usr/share/elasticsearch, -Des.path.conf=/usr/share/elasticsearch/config, -Des.distribution.flavor=default, -Des.distribution.type=docker, -Des.bundled_jdk=true]" }
{"type": "server", "timestamp": "2022-06-21T17:17:20,780Z", "level": "WARN", "component": "c.a.a.p.i.BasicProfileConfigFileLoader", "cluster.name": "elastdocker-cluster", "node.name": "elastdocker-node-0", "message": "Unable to load config file null", 
"stacktrace": ["java.security.AccessControlException: access denied (\"java.io.FilePermission\" \"/usr/share/elasticsearch/.aws/config\" \"read\")",
```

**Comment by thor0155** [comment]
i got same problem, and why it concern about aws, so can i disable it?

**Comment by mgeorgy** [comment]
This problem appears to be resolved by using the latest version of ES (8.10.2).
